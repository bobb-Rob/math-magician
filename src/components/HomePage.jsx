import React from 'react';

const HomePage = () => (
  <div>
    <h1>Welcome to our page!</h1>
    <p>
      Lorem ipsum dolor sit amet consectetur adipisicing elit.
      Natus et, ad at eum quam omnis, laboriosam sed possimus facere
      temporibus impedit labore voluptatibus tenetur fugiat vero aperiam
      eligendi, maiores dolores!
    </p>

    <p>
      Lorem ipsum dolor sit amet consectetur adipisicing elit.
      Natus et, ad at eum quam omnis, laboriosam sed possimus facere
      temporibus impedit labore voluptatibus tenetur fugiat vero aperiam
      eligendi, maiores dolores!
    </p>

  </div>
);

export default HomePage;
