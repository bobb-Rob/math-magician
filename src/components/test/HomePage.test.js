import React from 'react';
import renderer from 'react-test-renderer';
import HomePage from '../HomePage';

test('Test if homepage is renderred', () => {
  const homepage = renderer.create(<HomePage />).toJSON();
  expect(homepage).toMatchSnapshot();
});
