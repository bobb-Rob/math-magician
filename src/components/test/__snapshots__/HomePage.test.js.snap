// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test if homepage is renderred 1`] = `
<div
  className="home-page"
>
  <h2>
    Welcome to our page!
  </h2>
  <p>
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Natus et, ad at eum quam omnis, laboriosam sed possimus facere temporibus impedit labore voluptatibus tenetur fugiat vero aperiam eligendi, maiores dolores! Lorem ipsum dolor sit amet consectetur adipisicing elit. Natus et, ad at eum quam omnis, laboriosam sed possimus facere temporibus impedit labore voluptatibus tenetur fugiat vero aperiam eligendi, maiores dolores!
  </p>
  <br />
  <p>
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Natus et, ad at eum quam omnis, laboriosam sed possimus facere temporibus impedit labore voluptatibus tenetur fugiat vero aperiam eligendi, maiores dolores! Lorem ipsum dolor sit amet consectetur adipisicing elit. Natus et, ad at eum quam omnis, laboriosam sed possimus facere temporibus impedit labore voluptatibus tenetur fugiat vero aperiam eligendi, maiores dolores!
  </p>
</div>
`;
